frontend-react/cypress/support/commands/api.js

```diff
@@ -0,0 +1,61 @@
+import { GLOBAL_STORAGE_KEYS } from "../../../src/utils/SessionStorageTools";
+
+// NOTE: all of these commands expect cy.login() to have been called beforehand
+const accessToken = window.localStorage.getItem(
+    GLOBAL_STORAGE_KEYS.OKTA_ACCESS_TOKEN
+);
+
+const requestOptions = {
+    bearer: `Bearer ${accessToken}`,
+    timeout: 2 * 60 * 1000, // TODO: is two minutes enough?
+};
+
+export function fetchReceivers(organizationName) {
+    // TODO: consolidate with OrgReceiverSettingResource
+    return cy.request({
+        ...requestOptions,
+        url: `/api/settings/${organizationName}/receivers`,
+    });
+}
+
+export function fetchDeliveries(organizationName, receiverName) {
+    // TODO: consolidate with SubmissionsResource
+    // TODO: allow customizable query parameters
+    return cy.request({
+        ...requestOptions,
+        url: `/api/waters/org/${organizationName}.${receiverName}/deliveries?sortdir=DESC&cursor=3000-01-01T00:00:00.000Z&since=2000-01-01T00:00:00.000Z&until=3000-01-01T00:00:00.000Z&pageSize=61`,
+        auth: {
+            bearer: "Bearer ",
```

Comment:
Seems to be the only one using this `auth: { bearer: "..." }` property while the rest use the `requestOptions`. Intentional??

Reply:
Ah, this is very incomplete -- it should just be spreading `requestOptions` like the others :)